doc/user/content/overview/arrangements.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Arrangements"
+description: "Understand how Materialize arrangements work."
+menu:
+  main:
+    parent: 'overview'
+    weight: 4
+---
+
+We call the mechanisms that maintain materialized views for Materialize dataflows **arrangements**. Understanding them better can help you make decisions that will reduce memory usage while maintaining performance.
```

Comment:
Nit, but it isn't that we call them that. They were named this before MZ existed.